src/growth.jl

```diff
@@ -114,14 +115,14 @@ function ThresholdGrowth{R,W}(;
     ThresholdGrowth{R,W}(rate, threshold)
 end
 
-@inline function applyrule(data, rule::ThresholdGrowth, population, I)
-    intrinsicrate = get(data, rule.rate, I...)
-    threshold = get(data, rule.threshold, I...)
-    intrinsicrate >= threshold ? population : zero(population)
+@inline function applyrule(data, rule::ThresholdGrowth, N, I)
+    r = get(data, rule.rate, I...)
+    t = get(data, rule.threshold, I...)
+
+    return r >= t ? N : zero(N)
 end
```

Comment:
What is this rule used for?

Reply:
Cant remember now. It was just a way of masking everything below a threshold. Maybe we should just delete? this is pretty easy to write manually with `Cell`.

Reply:
Yeah, I think delete.

Reply:
I'll delete in another PR to keep this simple

Reply:
ok you're deleting. Just note that we should keep `t` for time, and so using `z` for threshold for instance. Other variables are less sensitive